test/e2e/testing-manifests/storage-csi/gce-pd/controller_ss.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
         - name: csi-external-provisioner
```

Comment:
make sure we're using the newest yamls here:
https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/deploy/kubernetes/base/controller.yaml

Reply:
Done